cmd/expiration-mailer/main.go

```diff
@@ -267,16 +268,18 @@ func (m *mailer) findExpiringCertificates() error {
 			)
 		} else {
 			_, err = m.dbMap.Select(
-				&certs,
-				`SELECT cert.* FROM certificates AS cert
-						 JOIN certificateStatus AS cs
-						 ON cs.serial = cert.serial
-						 AND cert.expires > :cutoffA
-						 AND cert.expires <= :cutoffB
-						 AND cs.status != "revoked"
-						 AND COALESCE(TIMESTAMPDIFF(SECOND, cs.lastExpirationNagSent, cert.expires) > :nagCutoff, 1)
-						 ORDER BY cert.expires ASC
-						 LIMIT :limit`,
+				&statuses,
+				`SELECT
```

Comment:
What is the formatting mismatch you're seeing? I restored the indent in https://github.com/letsencrypt/boulder/pull/2440/commits/f57c30133dfb0a3093247ad9da6b964af6900935 ?